### ISSUE TITLE
Error message should indicate which job trigger it

### ISSUE THREAD

**Issue by prymitive** [terinjokes/bakelite#1]
See https://travis-ci.org/cloudflare/unsee/jobs/328387786#L601
It's not very obvious which arch to blame for failures:
```
2018/01/13 07:08:29 fatal: There was an error! err='exit status 1' stdout='' stderr='vendor/github.com/spf13/viper/viper.go:34:2: build constraints exclude all Go files in /home/travis/gopath/src/github.com/cloudflare/unsee/vendor/github.com/fsnotify/fsnotify
```
Especially with high concurrency builds.

**Comment by terinjokes** [comment]
Thanks! I'll push some patches in the morning.

**Comment by terinjokes** [comment]
This should be fixed in [v0.0.4](https://github.com/terinjokes/bakelite/releases/tag/v0.0.4)
